src/Grids/conformal_cubed_sphere_face_grid.jl

```diff
@@ -0,0 +1,229 @@
+using CubedSphere
+using JLD2
+using OffsetArrays
+using Rotations
+
+using Oceananigans
+using Oceananigans.Grids
+using Oceananigans.Grids: AbstractGrid, R_Earth
+
+import Base: show
+
+struct ConformalCubedSphereFaceGrid{FT, TX, TY, TZ, A, R} <: AbstractGrid{FT, TX, TY, TZ}
+        Nx :: Int
+        Ny :: Int
+        Nz :: Int
+        Hx :: Int
+        Hy :: Int
+        Hz :: Int
+      λᶜᶜᵃ :: A
+      λᶠᶜᵃ :: A
+      λᶜᶠᵃ :: A
+      λᶠᶠᵃ :: A
+      φᶜᶜᵃ :: A
+      φᶠᶜᵃ :: A
+      φᶜᶠᵃ :: A
+      φᶠᶠᵃ :: A
+      zᵃᵃᶜ :: R
+      zᵃᵃᶠ :: R
+     Δxᶜᶜᵃ :: A
+     Δxᶠᶜᵃ :: A
+     Δxᶜᶠᵃ :: A
+     Δxᶠᶠᵃ :: A
+     Δyᶜᶜᵃ :: A
+     Δyᶜᶠᵃ :: A
+     Δyᶠᶜᵃ :: A
```

Comment:
I am not 100% sure the documentation lets someone know where Δyᶠᶜᵃ is. 

They can look at code, but are we sure that the suffix notation is written clearly, to avoid confusion?

Reply:
Ah are you saying that Δyᶠᶜᵃ is wrong/ambiguous for the cubed sphere? Or that we shouldn't be using it in general? I remember you suggested replacing it by Δyᶠᶠ⁻ (no superscript underscore so I used superscript minus) or maybe be explicit of the z location.

If it's a more general thing should we open a GitHub issue to discuss?

Reply:
Opening issue